src/terminal/input/mouseInput.cpp

```diff
@@ -93,6 +84,31 @@ static constexpr bool _isButtonDown(const unsigned int button) noexcept
     return isButtonDown;
 }
 
+// Routine Description:
+// - Retrieves which mouse button is currently pressed. This is needed because
+//      MOUSEMOVE events do not also tell us if any mouse buttons are pressed during the move.
+// Parameters:
+// <none>
+// Return value:
+// - a button corresponding to any pressed mouse buttons, else WM_LBUTTONUP if none are pressed.
+unsigned int TerminalInput::s_GetPressedButton() noexcept
+{
+    unsigned int button = WM_LBUTTONUP; // Will be treated as a release, or no button pressed.
+    if (WI_IsFlagSet(GetKeyState(VK_LBUTTON), KeyPressed))

```

Comment:
So, this isn't going to be awesome. This is a Win32 API that checks the Win32 view of the keyboard. It'll "work" in Terminal, but it is not appropriate for a shared component.

Reply:
+1. We should file a follow-up task to have the caller pass the mouse button state in to `HandleMouse`, though it'll probably work okay for now.